### PR TITLE
[0.9.x] KOGITO-1121 - Create advanced example for kogito + serverless workflow

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by tsurdilo** [comment]
If possible please merge https://github.com/kiegroup/kogito-runtimes/pull/439 first

File: serverless-workflow-service-calls-quarkus/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+```text
+mvn clean package 
+java -jar target/serverless-workflow-service-calls-quarkus-{version}-runner.jar    
+```
+
+or on windows
+
+```text
+mvn clean package
+java -jar target\serverless-workflow-service-calls-quarkus-{version}-runner.jar
```
Comment (ricardozanini) [inline serverless-workflow-service-calls-quarkus/README.md]: Maybe we could remove the version from the final jar in the maven plugin for clarity. We have done it in others examples.

**Review by ricardozanini — CHANGES_REQUESTED** [review]
Great Tiho! Just one small thing regarding the final name of the generated jar

File: serverless-workflow-service-calls-quarkus/README.md (hunk 1 above)
Comment (tsurdilo) [reply serverless-workflow-service-calls-quarkus/README.md]: fixed. thanks!

**Review by ricardozanini — APPROVED** [review]
Working beautifully:

Cool Serverless example

Please make sure to install this PR first: https://github.com/kiegroup/kogito-runtimes/pull/439

**Comment by cristianonicolai** [comment]
@tsurdilo this now has conflict

**Review by cristianonicolai — APPROVED** [review]

**Comment by tsurdilo** [comment]
@cristianonicolai rebased - thanks!

**Review by evacchi — APPROVED** [review]

**Review by MarianMacik — CHANGES_REQUESTED** [review]
We shouldn't merge this as this relies on kiegroup/kogito-runtimes#439

**Review by jstastny-cz — CHANGES_REQUESTED** [review]
Don't merge. We're past code-freeze.

**Comment by tsurdilo** [comment]
@MarianMacik @jstastny-cz :) wow....just wow guys....

**Comment by tsurdilo** [comment]
> Don't merge. We're past code-freeze.

@jstastny-cz is this still the case or can it be reviewed now please?

**Comment by tsurdilo** [comment]
@jstastny-cz updated tests to run in integration here as well.

**Review by jstastny-cz — APPROVED** [review]

**Review by MarianMacik — APPROVED** [review]

**Pull request merged**